packages/shared/src/components/modals/SquadMemberModal.tsx

```diff
@@ -91,6 +101,29 @@ export function SquadMemberModal({
             </SimpleTooltip>
           );
         }}
+        initialItem={
+          <button
+            type="button"
+            disabled={copying}
+            className="flex justify-start items-center py-3 px-6 hover:bg-theme-hover"
+            onClick={() => {
+              trackEvent({
+                event_name: AnalyticsEvent.ShareSquadInvitation,
+                extra: JSON.stringify({
+                  origin: Origin.SquadMembersList,
+                  squad: squad.id,
+                }),
+              });
+
+              copySquadInvite();
+            }}
+          >
+            <div className="flex justify-center items-center mr-4 w-12 h-12 bg-theme-float rounded-10">
+              <LinkIcon size={IconSize.Large} />
+            </div>
```

Comment:
I think we have a utility classed component like below.
```suggestion
            <FlexCentered className="mr-4 w-12 h-12 bg-theme-float rounded-10">
              <LinkIcon size={IconSize.Large} />
            </FlexCentered>
```